utils/vlog/vlog_to_pbtype.py

```diff
@@ -151,18 +151,24 @@ def make_pb_content(yj, mod, xml_parent, mod_pname, is_submode=False):
     """Build the pb_type content - child pb_types, timing and direct interconnect,
     but not IO. This may be put directly inside <pb_type>, or inside <mode>."""
 
-    def get_full_pin_name(pin):
+    def get_full_pin_name(pin, instance=-1):
```

Comment:
Probably it is worth using a macro such as `INVALID_INSTANCE = -1` globally accessible by the script to make it more understandable.

Reply:
makes sense

Reply:
done